src/config.js

```diff
@@ -0,0 +1,1602 @@
+
+/*
+ * decaffeinate suggestions:
+ * DS101: Remove unnecessary use of Array.from
+ * DS207: Consider shorter variations of null checks
+ * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
+ */
```

Comment:
don't forget to remove this block (:

Reply:
I saw in other decaffeinated files that people kept these in until they were addressed! Since there are still a lot of array.froms and null checks I left it in. 

I will fix those issues though and then delete this block!